### PR TITLE
Extend README with information about how to contribute.

### PR DESCRIPTION
**Pull request by brother**
Added a contribution section that touches on using a normal github
workflow with pull requests, issues and so on.
Also mention that all types of issues are welcome, security problems
are not supposed to be reported in an alternativ path.

These changes are part of qualification for the CII Best practices
badge.

### CODE REVIEW

**Comment by brother** [comment]
Progress: https://bestpractices.coreinfrastructure.org/en/projects/3288/

File: README.md [hunk 1]
```diff
@@ -313,3 +313,11 @@ composer require barryvdh/laravel-debugbar --dev
 When installed, make sure that `APP_DEBUG` is set to `true`, that's it.
 
 Customizing what data to collect and include in the response is easily done by copying the [default config file](https://github.com/barryvdh/laravel-debugbar/blob/master/config/debugbar.php) to `config/debugbar.php` and adjust as needed.
+
+## How To Contribute
+
+Development happens at github, any normal workflow with Pull Requests are welcom. In the sam spirit the Issue tracker at Github is used for any reports (regardless of the nature of the reports - feature requets, bugs of any nature and so on).
```
Comment (wecc) [inline README.md]:
```suggestion
Development happens at GitHub, any normal workflow with Pull Requests are welcome. In the same spirit the Issue tracker at GitHub is used for any reports (regardless of the nature of the reports - feature requests, bugs of any nature and so on).
```

File: README.md [hunk 2]
```diff
@@ -313,3 +313,11 @@ composer require barryvdh/laravel-debugbar --dev
 When installed, make sure that `APP_DEBUG` is set to `true`, that's it.
 
 Customizing what data to collect and include in the response is easily done by copying the [default config file](https://github.com/barryvdh/laravel-debugbar/blob/master/config/debugbar.php) to `config/debugbar.php` and adjust as needed.
+
+## How To Contribute
+
+Development happens at GitHub, any normal workflow with Pull Requests are welcome. In the same spirit the Issue tracker at GitHub is used for any reports (regardless of the nature of the reports - feature requets, bugs of any nature and so on).
```
Comment (christofferlindstrom) [inline README.md]: Verkar som om den här missades.
requets -> requests.

File: README.md [hunk 3]
```diff
@@ -313,3 +314,11 @@ composer require barryvdh/laravel-debugbar --dev
...
 
 Customizing what data to collect and include in the response is easily done by copying the [default config file](https://github.com/barryvdh/laravel-debugbar/blob/master/config/debugbar.php) to `config/debugbar.php` and adjust as needed.
+
+## How To Contribute
+
+Development happens at GitHub, any normal workflow with Pull Requests are welcome. In the same spirit the Issue tracker at GitHub is used for any reports (regardless of the nature of the reports - feature requests, bugs of any nature and so on).
+
+### Code standard
+
+As the library is intended for use in Laravel applications we encourage code standard to follow [https://laravel.com/docs/master/contributions#coding-style](upstream Laravel practices) - in short that would mean [https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md](PSR-2) and [https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader.md](PSR-4).
```
Comment (ttrig) [inline README.md]: Fel ordning på/i länkarna?

**Pull request merged**